gramps/gui/configure.py

```diff
@@ -1713,3 +1724,251 @@ def update_surn_height(self, obj, constant):
+                value = symbols.get_death_symbol_for_string(string)
+                font = fonts[idx]
+                fontname = font.family[0][1]
+                try:
+                    vals = all_fonts[fontname]
+                except:
+                    all_fonts[fontname] = []
+                if font.has_char(value):
+                    if value not in all_fonts[fontname]:
+                        all_fonts[fontname].append(value)
+                self.progress.step()
+        self.progress.close()
+        nb1 = 0
+        available_fonts = []
+        for font in all_fonts.keys():
+            font_usage = all_fonts[font]
+            if not font_usage:
+               continue
+            if len(font_usage) == nb_symbols: # If the font use all symbols
+                available_fonts.append(font)
+                nb1 += 1
+        config.set('utf8.available-fonts', available_fonts)
+        sel_font = config.get('utf8.selected-font')
+        active_val = 0
+        for val in available_fonts:
+            if sel_font == val:
+                break
+            active_val += 1
+        if len(available_fonts) > 0:
+            self.all_avail_fonts = [x for x in enumerate(available_fonts)]
+            self.add_combo(self.grid,
+                _('Choose font'),
+                5, 'utf8.selected-font',
+                self.all_avail_fonts, callback=self.utf8_update_font,
+                valueactive=True, setactive=active_val)
+        else:
+            self.add_text(self.grid,
+                _('You have no font with genealogical symbols on your '
+                  'system. Gramps will not be able to use symbols.'
+                 ),
+                4, line_wrap=True)
+            config.set('utf8.selected-font',"")
+        self.grid.show_all()
+        self.in_progress = False
+
+    def utf8_update_font(self, obj, constant):
+        entry = obj.get_active()
+        config.set(constant, self.all_avail_fonts[entry][1])
+        self.utf8_show_example()
+
+    def activate_change_font(self, obj=None):
+        if obj:
+            if not obj.get_active():
+                # reset to the system default
+                self.uistate.viewmanager.reset_font()
+        font = config.get('utf8.selected-font')
+        if not self.uistate.viewmanager.change_font(font):
+            # We can't change the font, so reset the checkbox.
+            if obj:
+                obj.set_active(False)
+
+    def utf8_show_example(self):
+        from gi.repository import Pango
+        from gramps.gen.utils.grampslocale import _LOCALE_NAMES as X
+        try:
+            # remove the old messages with old font
+            self.grid.remove_row(8)
+            self.grid.remove_row(7)
+        except:
+            pass
+        font = config.get('utf8.selected-font')
+        symbols = Symbols()
+        my_characters = _("What you will see") + " :\n"
+        my_characters += "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz "
```

Comment:
The `string` class can sometimes be useful, but maybe not worth the effort here:

`string.ascii_letters`

Reply:
Done